test/airportTestSuite.spec.js

```diff
@@ -0,0 +1,289 @@
+const { assertEquals } = require("./testing-framework");
+const Airport = require("../src/airport");
+const Plane = require("../src/plane");
+
```

Comment:
I think my tests are quite comprehensive! I am looking forward to using jasmine and a linter, as it took forever to format and clear all the tests.